### PR TITLE
Fix path to assemble-runtime script when it's inside of the runtime image

### PR DESCRIPTION
**Pull request by php-coder**
PTAL @bparees @PI-Victor

### CODE REVIEW

**Comment by php-coder** [comment]
Not fully tested yet.

File: pkg/build/strategies/sti/postexecutorstep.go [hunk 1]
```diff
@@ -251,7 +251,7 @@ func (step *startRuntimeImageAndUploadFilesStep) execute(ctx *postExecutorStepCo
 		return fmt.Errorf("Couldn't get working dir of %q image: %v", image, err)
 	}
 
-	commandBaseDir := workDir
+	commandBaseDir := filepath.Join(workDir, "scripts")
 	useExternalAssembleScript := step.builder.externalScripts[api.AssembleRuntime]
 	if !useExternalAssembleScript {
```
Comment (bparees) [inline pkg/build/strategies/sti/postexecutorstep.go]: i don't understand why this if-check is not being triggered for the scenario you described and setting commandBaseDir correctly so we run the script that is already part of the image?

File: pkg/build/strategies/sti/postexecutorstep.go [hunk 2]
```diff
@@ -266,7 +266,7 @@ func (step *startRuntimeImageAndUploadFilesStep) execute(ctx *postExecutorStepCo
 	}
 
 	cmd := fmt.Sprintf(
-		"while [ ! -f %q ]; do sleep 0.5; done; %s/scripts/%s; exit $?",
+		"while [ ! -f %q ]; do sleep 0.5; done; %s/%s; exit $?",
```
Comment (bparees) [inline pkg/build/strategies/sti/postexecutorstep.go]: nm, i understand now.

**Comment by bparees** [comment]
lgtm but let me know when you've tested it for both images w/ scripts, and repos that override the scripts.

**Comment by php-coder** [comment]
@bparees Fix is correct. Could you merge it, please?

**Comment by bparees** [comment]
[merge]

**Pull request merged**